src/python/pcmdi/__init__.py

```diff
@@ -1,4 +1,6 @@
 import io
 from mean_climate_metrics_calculations import compute_metrics
+from mean_climate_metrics_calculations_ExtInfo import compute_metrics_ExtInfo
```

Comment:
@doutriaux1 This is a mistake... sorry it's my first PR... The "Ext_Info" stuff has no reason to be here anymore. I don't even use it. If you prefer I can set up a new PR.